recipes/src/main/resources/quarkus-updates/org.apache.camel.quarkus/camel-quarkus/3alpha.yaml

```diff
@@ -25,6 +25,11 @@
 # Update the Camel - Quarkus extensions
 #####
 ---
+type: specs.openrewrite.org/v1beta/category
+name: Camel 3.x
+packageName: io.quarkus.updates.camel30
+description: Migrate from Camel 2.x to 3.x.
```

Comment:
The description should be `Migrate from Camel 3.x to 4.x.`

Reply:
```suggestion
description: Migrate from Camel 3.x to 4.x.
```